base/applications/regedit/treeview.c

```diff
@@ -795,24 +795,33 @@ BOOL SelectNode(HWND hwndTV, LPCWSTR keyPath)
 
     while(keyPath[0])
     {
+        size_t copyLength;
         s = wcschr(keyPath, L'\\');
-        lstrcpynW(szPathPart, keyPath, s ? s - keyPath + 1 : wcslen(keyPath) + 1);
+        if (s != NULL)
+        {
+            copyLength = (s - keyPath) * sizeof(WCHAR);
+        }
+        else
+        {
+            copyLength = sizeof(szPathPart);
+        }
```

Comment:
I am no fan of those 4 superfluous curly braces. Both: the command and the condition do fit into a single line and therefore they do not contribute to improved readability.. We are just increasing LOC-count here and reduce amount of code that can be viewed at the same time without scrolling.

Reply:
Maybe the best would be even to avoid the additional variable and just write it with the "?" operator again. That would save 9 lines of code. And would even "limit the scope" of the variable to the max by eliminating it.

Reply:
[Our coding style](https://reactos.org/wiki/Coding_Style#Braces) expressly permits braces around all blocks.